Restaging 52 (#4560)
EnterpriseDB/docs: product_docs/docs/pgd/5/routing/installing_proxy.mdx

PGD Proxy uses endpoints given in the local config file only at proxy startup. After that, PGD Proxy retrieves the list of actual endpoints (route_dsn) from the PGD Proxy catalog. Therefore, the node option `route_dsn` must be set for each PGD Proxy node. See [route_dsn](../routing#configuration) for more information.

##### Configuring health check

> PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check). If the health checks are required, you can enable them by setting the configuration parameters below to the pgd-proxy configuration file. By default it is disabled.

Reviewer: ```suggestion
PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check). If the health checks are required, you can enable them by adding the configuration parameters below to the pgd-proxy configuration file. By default it is disabled.
```

```yaml
cluster:
  name: cluster-name
  endpoints:
    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
  proxy:
    name: "proxy-a1"
    endpoint: "host=proxy-a1 port=6432 dbname=bdrdb user=pgdproxy "
    http:
      enable: true
      host: "0.0.0.0"
      port: 8080
      secure: false
      cert_file: ""
      key_file: ""
      probes:
        timeout: 10s
```

> It can be enabled by adding the config `cluster.proxy.http.enable: true`. When enabled, the HTTP server will be present on the default port, `8080`, with a ten second `timeout` and no HTTPS support.

Reviewer: ```suggestion
The API can be enabled by adding the config `cluster.proxy.http.enable: true`. When enabled, an HTTP server will listen on the default port, `8080`, with a ten second `timeout` and no HTTPS support.
```
